icaruscode/Timing/PMTTimingCorrectionsProvider.h

```diff
@@ -19,32 +19,24 @@
 #include "fhiclcpp/types/Sequence.h"
 #include "cetlib_except/exception.h"
 
-#include "lardata/DetectorInfoServices/DetectorClocksService.h"
-
+// Local
 #include "icaruscode/Timing/PMTTimingCorrections.h"
 
 // Database interface helpers
-#include "wda.h"
+#include "larevt/CalibrationDBI/Providers/DBFolder.h"
 
 // C/C++ standard libraries
-#include <memory> // std::unique_ptr<>
-#include <optional>
 #include <string>
-#include <utility> // std::move()
-#include <cassert>
-#include <tuple>
 
 namespace icarusDB::details {
     
-  struct PMTTimeCorrectionsDB {
-
-    double triggerCableDelay=0; ///< Expect nanoseconds.
-
-    double resetCableDelay=0; ///< Expect nanoseconds.
+  /// Structure for single channel corrections
+  struct PMTTimeCorrectionsDB { 
 
-    double laserCableDelay=0; ///< Expect nanoseconds.
-
-    double cosmicsCorrections=0; ///< Expect nanoseconds.
+    double triggerCableDelay=0;  ///< ns
+    double resetCableDelay=0;    ///< ns
+    double laserCableDelay=0;    ///< ns
+    double cosmicsCorrections=0; ///< ns
```

Comment:
Unless we store in the database in picoseconds, all these are actually microseconds:
```suggestion
    double triggerCableDelay=0;  ///< [&micro;s]
    double resetCableDelay=0;    ///< [&micro;s]
    double laserCableDelay=0;    ///< [&micro;s]
    double cosmicsCorrections=0; ///< [&micro;s]
```
(see e.g. `.cosmicsCorrections = -mean_residual_ns/1000` above).

Reply:
Point taken.